cmd/leader-elector/main.go

```diff
@@ -3,23 +3,25 @@ package main
 import (
 	"context"
 	"encoding/json"
-	"flag"
 	"net/http"
 	"os"
 	"os/signal"
 	"syscall"
 	"time"
 
+  "github.com/alexflint/go-arg"
```

Comment:
Indentation is crooked here :)

Reply:
Was due to my editor using spaces not tabs - I think I've fixed it all now!